src/game/game/server.rs

```diff
@@ -511,3 +523,18 @@ impl Handler<GameEndMessage> for GameServer {
         ctx.terminate();
     }
 }
+
+fn run_interval<F>(
+    ctx: &mut <GameServer as Actor>::Context,
+    duration: Duration,
+    closure: F
+)
+    where F: FnOnce(&mut GameServer, & <GameServer as Actor>::Context) -> Result<()> + 'static + Clone,
```

Comment:
Plutôt que d'ajouter `Clone` en contrainte, tu préférera changer le trait `FnOnce` en `FnMut`.
Ça te permettrait en plus de ne pas avoir à clone dans le code du dessous.

References pour comprendre la difference:
[FnMut in std::ops - Rust](https://doc.rust-lang.org/std/ops/trait.FnMut.html)
[FnOnce in std::ops - Rust](https://doc.rust-lang.org/std/ops/trait.FnOnce.html)

Reference pour comprendre les contraintes de trait de `run_interval` et `run_later`:
[AsyncContext in actix::prelude - Rust](https://docs.rs/actix/0.12.0/actix/prelude/trait.AsyncContext.html#method.run_interval)